.github/workflows/ci-tests/Examples_tests/test_launcher.sh

```diff
@@ -372,25 +450,36 @@ for dir in ./*/; do
 
 done # end non connected tests
 
-#****************************************************************************************************
-#*********************************** Start of Datc/s connected tests ********************************
-#****************************************************************************************************
+echo
+echo "****************************************************************************************************"
+echo "*********************************** Start of Datc/s connected tests ********************************"
+echo "****************************************************************************************************"
+echo
 
 erase_all_devices
 
 # Flash MAIN_DEVICE with BLE_datc
 cd $MSDK_DIR/Examples/$MAIN_DEVICE_NAME_UPPER/BLE_datc
+echo pwd=`pwd`
+echo
+
 make -j8
 
 # flash client first because it takes longer
 cd $MSDK_DIR/Examples/$MAIN_DEVICE_NAME_UPPER/BLE_datc/build
-printf "> Flashing BLE_datc on main device: $MAIN_DEVICE_NAME_UPPER\r\n "
-flash_with_openocd_fast $MAIN_DEVICE_NAME_LOWER $MAIN_DEVICE_ID 1
+printf "\r\n> Flashing BLE_datc on main device: $MAIN_DEVICE_NAME_UPPER\r\n"
+#flash_with_openocd_fast $MAIN_DEVICE_NAME_LOWER $MAIN_DEVICE_ID 1
+flash_with_openocd $MAIN_DEVICE_NAME_LOWER $MAIN_DEVICE_ID 1
 
 # flash DUT with BLE_dats
 cd $MSDK_DIR/Examples/$DUT_NAME_UPPER/BLE_dats/build
-printf "> Flashing BLE_dats on DUT $DUT_NAME_UPPER \r\n"
-flash_with_openocd_fast $DUT_NAME_LOWER $DUT_ID 2
+printf "\r\n> Flashing BLE_dats on DUT $DUT_NAME_UPPER\r\n"
+#flash_with_openocd_fast $DUT_NAME_LOWER $DUT_ID 2
+flash_with_openocd $DUT_NAME_LOWER $DUT_ID 2
+
+# Reset the two boards
+reset_board_by_openocd max32655 $MAIN_DEVICE_NAME_LOWER $MAIN_DEVICE_ID
+reset_board_by_openocd max32655 $DUT_NAME_UPPER         $DUT_ID
```

Comment:
i think the boards failed to pair because when resetting the log says it failed to find device, here we are passing upper case for one and lower case for the other. 

```
2023-01-24T19:13:10.1649041Z function: reset_board_by_openocd max32655 MAX32665 0409000098d9439b00000000000000000000000097969906
2023-01-24T19:13:10.1699256Z Open On-Chip Debugger 0.11.0+dev-g124409881-dirty (2022-08-15-16:03)
2023-01-24T19:13:10.1700138Z Licensed under GNU GPL v2
2023-01-24T19:13:10.1700680Z For bug reports, read
2023-01-24T19:13:10.1701316Z 	http://openocd.org/doc/doxygen/bugs.html
2023-01-24T19:13:10.2397772Z Error: unable to find a matching CMSIS-DAP device
```